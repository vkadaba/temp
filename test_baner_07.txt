{
    c = + < d e < a b;
    d = 4.5;
    e = true;
}
