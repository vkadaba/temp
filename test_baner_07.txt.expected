TYPE MISMATCH 4 C1
